fsspec/implementations/dropboxdrivefs.py

```diff
@@ -0,0 +1,244 @@
+import requests
+import dropbox
+from ..spec import AbstractFileSystem, AbstractBufferedFile
+
+
+class DropboxDriveFileSystem(AbstractFileSystem):
+    """ Interface dropbox to connect, list and manage files
+    Parameters:
+    ----------
+    token : str
+          Generated key by adding a dropbox app in the user dropbox account. 
+          Needs to be done by the user
+
+    """
+
+    def __init__(self, **storage_options):
```

Comment:
A bit clearer to have token as a required argument I think.

```suggestion
    def __init__(self, token, *args, **storage_options):
```

but still pass it through to `super()`.